docker/docker-start.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+mkdocs serve -a 0.0.0.0:8000
```

Comment:
Why not `127.0.0.1:8000`? To avoid the warning:
``` bash
WARNING -  Config value 'dev_addr': The use of the IP address '0.0.0.0' suggests a production environment or the use of a proxy to connect to the MkDocs server. However, the MkDocs' server is intended for local development purposes only. Please use a third party production-re
```

Reply:
Because otherwise mkdocs only listens on requests on localhost coming from inside the container. The warning is okay, because we use docker only for local development purposes, not for a production environment.